### DISCUSSION TITLE
[Feature Request] Terrain Material Traction/Torque Multipliers

### DISCUSSION THREAD

**Discussion by postmvn** [SmartlyDressedGames/Unturned-3.x-Community#4664] [Ideas]
Not sure if it was proposed before, but it seems **alot** of games have this feature.  It would finally solve good ol' issue of sportscars offroading on the same level as specialized offroad vehicles. Decreasing torque would be a much simplier task I think. Decreasing friction   could not give any result as many vehicle mods use custom physics and results would vary alot. 

--

**Option 1**
I'd imagine something like different asset with blade/caliber system but for terrain materials with different multipliers in vehicle dat files.
That way this function would be pretty much most configurable. But more work :)

--

**Option 2**
Just a simple system with two torque multipliers which are Offroad / Road
No idea how it could be set up for everything on map but for the first iteration it could be:

**Road:**
-All roads (not sure if its possible to configure different road types to have different traction type)
-All objects

**Offroad:**
-Grass/Dirt/Snow/Sand terrain materials

--

**Vehicle Configuration:**

**Traction** can be left as legacy function or replaced with modifyable function like Traction_Blizzard 0.3 when its blizzard.

Those multipliers would be applied to the final torque value of each wheel if its on one of the two types of terrain:

**Traction_Road 1 
Traction_Offroad 0.5** 

(hopefully it would be possible to make it more than 1 to use on vehicles that perform offroad better than on roads)

This multiplier would be applied before all previous multipliers if blizzard is present:

**Traction_Blizzard 0.5**

Pardon for mistakes if there are any, had to rush it a bit :)

**Comment by SDGNelson** [comment]
I agree that per-surface tire friction/traction will be necessary. Aiming to do this as part of revising tire friction across the board!
